### PR TITLE
[HCICD-462] : Notifications to Teams

### PR DESCRIPTION
**Pull request by AdityaHitachi**
Added Teams Notify action to send messages to a Microsoft Teams channel to help users keep track of all the defined steps in a workflow, along with additional workflow details.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -66,6 +66,20 @@ List of Composite Actions:
 
 8. Tag: Commit the changes available in the workspace and tag the code as per the latest commit id. You have also the possibility of pushing to a tag only, by setting `push_tag_only` to `true`.
 
+9. Notification: Aims to send messages to a Microsoft Teams channel and slack channel to help users keep track of all the defined steps in a workflow, along with additional workflow details.
```
Comment (cardosov) [inline README.md:69]: The reporting step also writes to the summary. 

When you say
> Aims to send messages to a Microsoft Teams channel and slack channel

it should be `Aims to send messages to a Microsoft Teams channel and/or a slack channel ...`

**Review by cardosov — CHANGES_REQUESTED** [review]

File: README.md (hunk 1 above)
Comment (AdityaHitachi) [reply README.md]: Updated.
Also Added tag `1.0.0` for team-notify-action

Comment (cardosov) [reply README.md]: I don't see any reference to the Summary part when doing the report step. And call it `Reporting` instead of `Notification`

Comment (AdityaHitachi) [reply README.md]: Updated the Reporting section

**Review by cardosov — APPROVED** [review]

**Review by vganpisetty — APPROVED** [review]

**Pull request merged**